parent/pom.xml

```diff
@@ -1482,5 +1491,66 @@
         </plugins>
       </build>
     </profile>
+
+    <!-- profile to disable the fmt-plugin on older jdks -->
```

Comment:
Nit: I'd argue it's to enable formatting on jdks 9 and above :sweat_smile:

Reply:
No, it's to enable formatting using v 2.10+, which is incompatible with jdks < 9. so we disable the plugin: `<phase>none</phase>`

```suggestion
    <!-- profile to disable the fmt-plugin on unsupported jdks -->
```

Reply:
True, I totally misread that :sweat_smile: